### PR TITLE
Update all machines se050 support

### PR DESCRIPTION
**Pull request by vanmaegima**

### CODE REVIEW

**Comment by doanac** [comment]
Docs for b55f43cd105333472ab8e72048541b27d60c9205 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1147/docs/artifacts/html/index.html

File: source/reference-manual/boards/se050_imx6ull.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
+i.MX 6ULL Evaluation Kit with SE050ARD
+========================================
+
+This document will walk a developer through the steps of installing a
+FoundriesFactory image with the SE050 hardware enabled onto the NXP
+imx6ullevk that is connected to the NXP OM-SE050ARD development platform.
```
Comment (angolini) [inline source/reference-manual/boards/se050_imx6ull.rst:6]:
```suggestion
`imx6ullevk` that is connected to the NXP OM-SE050ARD development platform.
```

**Review by angolini — APPROVED** [review]
LGTM, only minor change

**Comment by doanac** [comment]
Docs for a894fab49d4c927614d4d16e168b3f73c8628c16 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1149/docs/artifacts/html/index.html

**Review by Tim-Anderson — APPROVED** [review]
The PR title says 6ull but you have modified 8mm and 8mq. maybe add update all machines se050 support.

Otherwise LGTM

**Review by ricardosalveti — APPROVED** [review]
LGTM, thanks for working on this.

**Pull request merged**